serving/backend/server.py

```diff
@@ -0,0 +1,27 @@
+import joblib
+from api_models import Properties
+from calculate_level import calculate_level
+from fastapi import FastAPI
+from starlette.middleware.cors import CORSMiddleware
+
+
+app = FastAPI()
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=["*"],
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
+
+model = joblib.load(filename="../../saved_models/current_model.pkl")
+
+
+@app.post("/properties/upload")
+async def upload_properties(properties: Properties):
```

Comment:
Tylko skoro nic nie upload'ujemy, to można by zmienić nazwe endpointu, nie? Tzn. ten endpoint po prostu robi predykcję, może coś w stylu `make_prediction`?